swan-lake/learn-the-language/distinctive-language-features/concurrency.md

```diff
@@ -506,26 +506,20 @@ In  Ballerina, you have a ``readonly`` type that represents immutable values. Th
 // The type of `s` is an immutable `string`` array.
 // The value is also constructed as an immutable value.
 readonly & string[] s = ["foo", "bar"];
- 
-type Row record {
-    // Both the field and its value are immutable.
-    readonly string[] k;
 
-    int v;
-};
+isolated function foo(string[] s) {
+    
+}
 
-table<Row> key(k) t = table [
-    // Can safely use `s` as a key.
-    {k: s, v: 17}
-];
+public function main() returns error? {
+    foo(s);
+}
 ```
 
 In the above code example, the string array **``s``** is declared as read-only using the type intersection operator ``&``. The type of **``s``** is both an array of strings and ``readonly,`` which means that it is an immutable array. This is enforced at compile-time to ensure that the values of the array **``s``** are immutable. Therefore, it is safe to pass it as an argument to an ``isolated`` function.
 
 This concept is different from the const keyword in C. So a *const char s* in C is not the same as *readonly & string[] s* in Ballerina, because here you are making the values of the type immutable whereas, in the case of C the const identifier refers to a variable whose value can be assigned to another variable with the const qualifier.
 
-If the ``readonly`` keyword is used within a structured type, it makes both the field and the value immutable. In the above code example, the field **``k``** of the ``Row`` record type has the ``readonly`` qualifier. This means that the field **``k``** cannot be assigned a new value, and the values of the field cannot be mutated by changing the array **``s``**.
```

Comment:
This change is not right. We are removing important content. We can change the example instead.

Reply:
Added this